File: 351001/orsik/publisher/src/main/java/com/sergey/orsik/dto/response/CreatorResponseTo.java
package com.sergey.orsik.dto.response;

import com.sergey.orsik.entity.CreatorRole;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class CreatorResponseTo {

    private Long id;
    private String login;
    private String firstname;
    private String lastname;
    private CreatorRole role;
}

File: 351002/vorobei/Publisher/BusinessLogic/DTO/Response/CreatorResponseTo.cs
using BusinessLogic.DTO.Request;
using DataAccess.Models;
using System.Text.Json.Serialization;

namespace BusinessLogic.DTO.Response
{
    public class CreatorResponseTo : BaseEntity
    {
        [JsonPropertyName("login")]
        public string Login { get; set; } = string.Empty;

        [JsonPropertyName("password")]
        public string Password { get; set; } = string.Empty;

        [JsonPropertyName("firstname")]
        public string FirstName { get; set; } = string.Empty;

        [JsonPropertyName("lastname")]
        public string LastName { get; set; } = string.Empty;

        [JsonPropertyName("role")]
        public string Role { get; set; } = string.Empty;
    }
}